### ISSUE TITLE
Consider migration to NEAR Lake

### ISSUE THREAD

**Issue by frol** [octopus-network/octopus-near-indexer-kafka#1]
I feel you may drop Kafka altogether (we actually originally considered to use Kafka as intermediate storage as well) and just use S3 storage directly, see the announcement: https://gov.near.org/t/announcement-near-lake-framework-brand-new-word-in-indexer-building-approach/17668

**Comment by zhang-wenchao** [comment]
You are right, our internal project has given up Kafka Streams and KSQL, which we will continue to maintain, because we believe that there will be many users will need it, follow we will cooperate with Near-Lake, Indexer -> S3 -> Kafka.

**Comment by zhang-wenchao** [comment]
We will announce roadmap and support in the near future and hope to make more contributions to NEAR ecology.
